dev-python/logfury/logfury-0.1.2.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="Boilerplate library for logging method calls"
+HOMEPAGE="https://github.com/ppolewicz/logfury https://pypi.org/project/logfury/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	virtual/python-funcsigs[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}] "
```

Comment:
non-conditional deps should be sorted  alphabetically.
Also remove the whitespace before the closing `"`

Also, RDEPEND is missing.
DEPEND != RDEPEND by default if EAPI=6
https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-650007.3.3

you should probably add six and funcsigs to RDEPEND,
and add `${RDEPEND}` to `DEPEND` along with setuptools.

check out for example dev-python/PyGithub/PyGithub-1.40_alpha4.ebuild how to do that. I just picked random ebuild that can hint you.